### DISCUSSION TITLE
Upgrader better in modal?

### DISCUSSION THREAD

**Discussion by twMat** [TiddlyWiki/TiddlyWiki5#6525] [Ideas]
I'm typically no fan of modals but I think it would make sense in the **upgrader** because the users stuff interferes with the UI after the "tiddler transfer" before the saving of the upgraded wiki. Here is a screendump showing one case but I've had it happen many times.

<img width="959" alt="image" src="https://user-images.githubusercontent.com/4534056/158014696-1e6600ad-21c4-41d1-b76d-ed962281c83b.png">

**Comment by Jermolene** [comment]
Hi @twMat

> the users stuff interferes with the UI after the "tiddler transfer" before the saving of the upgraded wiki

That's definitely an annoying problem. It's a long time since the upgrader received any attention, and now it would probably make sense to rework it to have a completely custom page layout (like we do for https://tiddlywiki.fission.app/).

The deeper problem is the way that the upgrader has to fully load all of the tiddlers of the wiki, which means that all customisations are applied to the upgrader. It may be feasible by now to do things differently, without having to fully load the tiddlers.
